bedrock/firefox/templates/firefox/onboarding/user-actions-pinsearch.html

```diff
@@ -0,0 +1,48 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% extends "firefox/onboarding/onboarding-base.html" %}
+
+{% block page_css %}
+  {% stylesheet 'firefox_onboarding_user_actions' %}
+{% endblock %}
+
+{% block title %}
+  {{ _('Ready, set, Firefox!') }}
+{% endblock %}
+
+{% block onboarding_content %}
+  <ul id="user-actions">
+    <li class="user-action pinit">
+      <h3>{{ _('Keep Firefox close') }}</h3>
+
+      {# Default instructions - non-OS X #}
+      <ul class="instructions pinit-other">
+        <li>{{ _('Right-click the Firefox icon below') }}</li>
+        <li>{{ _('Click the pin to keep it there') }}</li>
+      </ul>
+
+      {# OS X specific instructions #}
+      <ul class="instructions pinit-mac">
+        <li>{{ _('Click and hold the Firefox icon below') }}</li>
```

Comment:
Hmm, maybe I didn't check closely enough, but would it not create duplicate `twinkle` animation rules?